tools/topology2/abi-generated.conf

```diff
@@ -0,0 +1,5 @@
+Object.manifest."sof_manifest" {
+	data."sof_manifest" {
+		bytes	"0x03,0x12,0x01"
+	}
+}
```

Comment:
@lgirdwood this is a generated file. Ideally it should go into the build folder. But because alsatplg only allows 1 base include folder, I had to generate it here. Nevertheless, this shouldnt  be in the PR let me remove it. And yes, this will be private data in the manifest